db/migrate_cards/20150907173833_import_production_export.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class ImportProductionExport < Card::Migration
+  def up
+    import_json "production_export.json"
+    
+  end
+end
```

Comment:
this is just to create three cards.
1. production_export pointer card that contains all needed card names only
2.  two search cards that get all cards with codename and type as cardtype